README.md

```diff
@@ -96,61 +93,67 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
 
 To start using the SDK in your applications, create a Client instance and then invoke the API on it:
 
-#### With login ID and password
+#### Define connection parameters
 
-```python3
-#!/usr/bin/env python3
+In order to login to conjur you need to have 5 parameters known from advance.
 
-from conjur import Client
+```
```

Comment:
this is just a list of what the need, shouldn't include values - just a table with paraneter and what it is
for example
| Parameter | Description |
|---|---|
| conjur_url | The Conjur URL, for example http://mrorg.example.com |

Reply:
and so on

Reply:
how do you left-align a column heading?

Reply:
Alternaivelt, say "To log in to Conjur, define the following parameters, for example: